src/main/java/ru/hse/application/services/DoctorService.java

```diff
@@ -0,0 +1,31 @@
+package ru.hse.application.services;
+
+import ru.hse.application.data.DoctorDatabase;
+import ru.hse.application.models.Doctor;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DoctorService {
+    private final static Pattern pattern = Pattern.compile("^[a-z A-Z]+$");
```

Comment:
:+1:

Reply:
BTW, just a notice -- mostly likely it would be faster to check whether word is only Latin Characters via lambda instead of Regular Expression.